### PR TITLE
CPDTP-463: Update participant action docs

### PR DESCRIPTION
**Pull request by kicferk1**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDTP-463

Also fixes imperfect filter documentation in swagger

### Is there anything that the code reviewer should know?

The guidance was getting lengthy, so I started using partials. They have no variables or anything complex, so I don't think we need to componentise them. 

### Code quality checks
- [x] All commit messages are meaningful and true
- [x] Added enough automated tests

### HTML Checks
- [x] All new pages have automated accessibility checks
- [x] All new pages have visual tests via Percy

### Gotchas
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Go to LP ecf guidance page

## External API changes

Does this change anything in our external APIs? If so, did you remember to update the CHANGELOG for Lead Providers? (lead-providers/guidance/release-notes) - no api changes

### CODE REVIEW

**Comment by jamesrose** [comment]
@kicferk1 Could you update this to include https://github.com/DFE-Digital/early-careers-framework/pull/984 once it's merged?

**Comment by kicferk1** [comment]
> @kicferk1 Could you update this to include #984 once it's merged?

Will do :)

**Review by jamesrose — APPROVED** [review]

File: swagger/v1/component_schemas/ListFilter.yml [hunk 1]
```diff
@@ -5,7 +5,3 @@ properties:
     description: "Return only records that have been updated since this date and time (ISO 8601 date format)"
     type: :string
     example: "2021-05-13T11:21:55Z"
-  participant_id:
```
Comment (jamesrose) [inline swagger/v1/component_schemas/ListFilter.yml:8]: We removed `participant_id` here but `GET /api/v1/participant-declarations` can actually filter by it. Do you need another component schema for this one specifically?

Comment (kicferk1) [reply swagger/v1/component_schemas/ListFilter.yml:8]: I do, that's what `swagger/v1/component_schemas/ListFilterDeclarations.yml` is for :)

**Review by jamesrose — APPROVED** [review]

**Pull request merged**